capstone/src/main/java/com/google/univiz/CarnegieSizeDegree.java

```diff
@@ -0,0 +1,34 @@
+package com.google.univiz;
+
+/**
+ * CarnegieSizeDegree is a class that holds the enum for the range of values for a school's carnegie
+ * size degree. The index of each of these values is the carnegie size degree provided by
+ * ScorecardData.
```

Comment:
Since this is in UniViz, let's leave scorecard details out...

```suggestion
 * size degree.
```